src/runtime/config.h

```diff
@@ -82,12 +87,14 @@ struct UserConfigPolicy {
   static constexpr bool allowed_in_userconfig = false;
   type user_config;
   static constexpr type UserConfigPolicy::*value = &UserConfigPolicy::user_config;
-  static constexpr Override<input_type> override_value = nullptr;
+  static constexpr Override<input_type> override_value = &WakeConfigOverrides::user_config;
+  static constexpr const char* env_var = "WAKE_USER_CONFIG";
```

Comment:
Can you update  the  config tests? I'd like to see a test where a value is sourced from the env